homeassistant/components/teslemetry/entity.py

```diff
@@ -73,79 +129,81 @@ async def wake_up_if_asleep(self) -> None:
                         raise HomeAssistantError("Could not wake up vehicle")
                     await asyncio.sleep(times * 5)
 
-    def get(self, key: str | None = None, default: Any | None = None) -> Any:
-        """Return a specific value from coordinator data."""
-        return self.coordinator.data.get(key or self.key, default)
-
-    def set(self, *args: Any) -> None:
-        """Set a value in coordinator data."""
-        for key, value in args:
-            self.coordinator.data[key] = value
-        self.async_write_ha_state()
+    async def handle_command(self, command) -> dict[str, Any]:
```

Comment:
I think this could be good to have as a decorator instead on the respective methods that's using it (in a follow-up PR)